backend/src/test/java/org/eclipse/tractusx/semantics/registry/JwtTokenFactory.java

```diff
@@ -12,149 +12,166 @@
-        String tenantId;
-
-        public RequestPostProcessor allRoles(){
-            return authenticationWithRoles(tenantId,
-                    List.of(AuthorizationEvaluator.Roles.ROLE_VIEW_DIGITAL_TWIN,
-                            AuthorizationEvaluator.Roles.ROLE_ADD_DIGITAL_TWIN,
-                            AuthorizationEvaluator.Roles.ROLE_UPDATE_DIGITAL_TWIN,
-                            AuthorizationEvaluator.Roles.ROLE_DELETE_DIGITAL_TWIN)
-            );
-        }
-
-        public RequestPostProcessor readTwin(){
-            return authenticationWithRoles(tenantId, List.of(AuthorizationEvaluator.Roles.ROLE_VIEW_DIGITAL_TWIN));
-        }
-
-        public RequestPostProcessor addTwin(){
-            return authenticationWithRoles(tenantId, List.of(AuthorizationEvaluator.Roles.ROLE_ADD_DIGITAL_TWIN));
-        }
-
-        public RequestPostProcessor updateTwin(){
-            return authenticationWithRoles(tenantId,List.of(AuthorizationEvaluator.Roles.ROLE_UPDATE_DIGITAL_TWIN));
-        }
-
-        public RequestPostProcessor deleteTwin(){
-            return authenticationWithRoles(tenantId, List.of(AuthorizationEvaluator.Roles.ROLE_DELETE_DIGITAL_TWIN));
-        }
-
-        public RequestPostProcessor withoutResourceAccess(){
-            Jwt jwt = Jwt.withTokenValue("token")
-                    .header("alg", "none")
-                    .claim("sub", "user")
-                    .build();
-            Collection<GrantedAuthority> authorities = Collections.emptyList();
-            return authentication(new JwtAuthenticationToken(jwt, authorities));
-        }
-
-        public RequestPostProcessor withoutRoles(){
-            Jwt jwt = Jwt.withTokenValue("token")
-                    .header("alg", "none")
-                    .claim("sub", "user")
-                    .claim("resource_access", Map.of(publicClientId, new HashMap<String, String>()))
-                    .build();
-            Collection<GrantedAuthority> authorities = Collections.emptyList();
-            return authentication(new JwtAuthenticationToken(jwt, authorities));
-        }
-
-        private RequestPostProcessor authenticationWithRoles(String tenantId, List<String> roles){
-            Jwt jwt = Jwt.withTokenValue("token")
-                    .header("alg", "none")
-                    .claim("sub", "user")
-                    .claim("resource_access", Map.of(publicClientId, Map.of("roles", toJsonArray(roles) )))
-                    .build();
-            Collection<GrantedAuthority> authorities = Collections.emptyList();
-            return authentication(new JwtAuthenticationToken(jwt, authorities));
-        }
-
-        private static JSONArray toJsonArray(List<String> elements){
-            JSONArray jsonArray = new JSONArray();
-            for (String element : elements){
-                jsonArray.appendElement(element);
-            }
-            return jsonArray;
-        }
-    }
+public class JwtTokenFactory
+{
```

Comment:
Please avoid reformats changing the code style (block should not start in new line, the project uses 3 spaces instead of tabs, etc.).

Reply:
As already written, do you have a template for Eclipse to do the code formatting as you will expect?
We have a company internal formatting template and I just used that, because it is set per default in my environment.

Reply:
Thanks, I have added a comment in my response here: https://github.com/eclipse-tractusx/sldt-digital-twin-registry/pull/290#issuecomment-2063328141